storage/store.go

```diff
@@ -472,6 +472,7 @@ func (s *Store) Start(stopper *util.Stopper) error {
 	if s.ctx.Gossip != nil {
 		s.ctx.Gossip.RegisterCallback(gossip.KeyConfigAccounting, s.configGossipUpdate)
 		s.ctx.Gossip.RegisterCallback(gossip.KeyConfigZone, s.configGossipUpdate)
+		s.ctx.Gossip.RegisterCallback(gossip.KeyConfigPermission, s.configGossipUpdate)
```

Comment:
It is intentional. These callbacks are registered in order to respond quickly to config changes which require us to split ranges. We split ranges if the range spans a boundary for certain config types. The two which this is true for are accounting (we want to just be able to add up range accounting stats without having to figure out which part of the range counts towards one accounting prefix and which counts towards an adjacent accounting prefix. Same thing is true with zone configs, but even more so. Zone configs can specify that the ranges which fall within them have N replicas, or belong to which datacenters, or split at how many bytes. Obviously we can't have a range which straddles two different zone configs.

Permissions, on the other hand, don't have any of these constraints, so we don't need to split ranges on permission config prefix boundaries.